lib/erb_lint/linters/final_newline.rb

```diff
@@ -10,22 +10,22 @@ def initialize(config)
         @new_lines_should_be_present = config['present'].nil? ? true : config['present']
       end
 
-      protected
-
-      def lint_lines(lines)
+      def lint_file(file_tree)
         errors = []
-        return errors if lines.empty?
+        return errors if file_tree.children.size == 1
```

Comment:
What’s another name for `file_tree.children.size == 1`? Can you turn that into a named method so it’s clear what that circumstance is called?